tests/integration_tests/datasets/api_tests.py

```diff
@@ -2255,6 +2274,8 @@ def test_duplicate_virtual_dataset(self):
         assert len(new_dataset.columns) == 2
         assert new_dataset.columns[0].column_name == "id"
         assert new_dataset.columns[1].column_name == "name"
+        db.session.delete(new_dataset)
+        db.session.commit()
```

Comment:
This test didn't clean up after itself, causing my new IT to fail, hence the added cleanup here